cmd/url.go

```diff
@@ -36,14 +38,14 @@ var urlCreateCmd = &cobra.Command{
 
 The created URL can be used to access the specified component from outside the OpenShift cluster.
 `,
-	Example: `  # Create a URL for the current component.
-  odo url create
+	Example: `  # Create a URL for the current component with a specific port
+  odo url create --port 8080
```

Comment:
we shouldn't have `--port` flag in every example, it looks like it is mandatory argument.